upcloud_api/cloud_manager/storage_mixin.py

```diff
@@ -41,10 +41,10 @@ def get_storage(self, storage: str) -> Storage:
 
     def create_storage(
         self,
+        zone: str,
         size: int = 10,
         tier: str = 'maxiops',
         title: str = 'Storage disk',
```

Comment:
It would probably be better to make these [keyword-only arguments](https://www.python.org/dev/peps/pep-3102/) using the lone star (sorry, the suggestion doesn't directly apply since it'd need to be over a deleted line, which is not supported by GitHub):

```suggestion
        self,
        *,
        zone: str,
        size: int = 10,
        tier: str = 'maxiops',
        title: str = 'Storage disk',
```